### DISCUSSION TITLE
Livewire 3 when I have a wire:model.live and #[Modelable] the value doesn't update "live" on parent

### DISCUSSION THREAD

**Discussion by sektormediaofficial** [livewire/livewire#7026] [7. Bugs (archived)]
### Livewire version

3.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a parent component
```php
<?php
 
namespace App\Livewire;
 
use Illuminate\Support\Facades\Auth;
use Livewire\Component;
use App\Models\Todo;
 
class TodoList extends Component
{
    public $todo = '';
 
    public function add()
    {
        Todo::create([
            'content' => $this->todo,
        ]);
 
        $this->reset('todo');
    }
 
    public function render()
    {
        return view('livewire.todo-list', [
            'todos' => Auth::user()->todos,
        ]);
    }
}
```

2. Create a child component with #[Modelable]
```php
<?php
 
namespace App\Livewire;
 
use Livewire\Component;
use Livewire\Attributes\Modelable;
 
class TodoInput extends Component
{
    #[Modelable] 
    public $value = '';
 
    public function render()
    {
        return view('livewire.todo-input');
    }
}
```

3. add `wire:model.live` to both parent and child
```blade
// child
<div>
    <p>Value is: {{ $value }}</p>
    <input type="text" wire:model.live="value" >
</div>


// parent
<div>
  <livewire:todo-input wire:model.live="todo" /> 
   <p>Value is: {{ $todo }}</p>
</div>
```

Bug: The parent's value of "todo" Won't update in HTML until you try to submit a form. Is this intentional?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by papa-ree** [comment]
you can use Computed properties for alternative
